src/main/kotlin/com/sourcegraph/cody/chat/ui/SingleMessagePanel.kt

```diff
@@ -1,21 +1,23 @@
-package com.sourcegraph.cody.chat
+package com.sourcegraph.cody.chat.ui
 
 import com.intellij.openapi.project.Project
 import com.intellij.ui.ColorUtil
 import com.intellij.util.ui.SwingHelper
 import com.intellij.util.ui.UIUtil
 import com.sourcegraph.cody.agent.protocol.ChatMessage
 import com.sourcegraph.cody.agent.protocol.Speaker
+import com.sourcegraph.cody.chat.*
 import com.sourcegraph.cody.ui.HtmlViewer.createHtmlViewer
-import java.awt.Color
-import javax.swing.JEditorPane
-import javax.swing.JPanel
 import org.commonmark.ext.gfm.tables.TablesExtension
 import org.commonmark.node.Node
 import org.commonmark.parser.Parser
 import org.commonmark.renderer.html.HtmlRenderer
+import java.awt.Color
+import java.util.*
+import javax.swing.JEditorPane
+import javax.swing.JPanel
 
-class MessagePanel(
+class SingleMessagePanel(
```

Comment:
why is it "Single"?

Reply:
I could just leave it named `MessagePanel` but there is also `MessagesPanel` class (maybe I should name that one other way) and for my taste it was too easy to mistake those two. I wanted to stress the fact that this class wraps just a single message.